sdk-tests/src/test/java/io/dapr/it/state/AbstractStateClientIT.java

```diff
@@ -109,12 +109,12 @@ public void saveAndGetBulkStates() {
 
     assertEquals(stateKeyTwo, result.stream().skip(1).findFirst().get().getKey());
     assertNull(result.stream().skip(1).findFirst().get().getValue());
-    assertEquals("1", result.stream().skip(1).findFirst().get().getEtag());
+    assertNull(result.stream().skip(1).findFirst().get().getEtag());
```

Comment:
@artursouza I wonder how this assert was successful before.

Reply:
Ops. Let me see if this is a bug that was fixed or created.

Reply:
I see that his change is not in anymore. So I assume it was a red herring.